yarn-project/end-to-end/src/e2e_event_logs.test.ts

```diff
@@ -106,59 +41,57 @@ describe('Logs', () => {
       const lastBlockNumber = Math.max(...txs.map(tx => tx.blockNumber!));
       const numBlocks = lastBlockNumber - firstBlockNumber + 1;
 
-      // We get all the events we can decrypt with our incoming viewing keys
-
+      // Each emit_encrypted_events call emits 2 ExampleEvent0s and 1 ExampleEvent1
+      // So with 5 calls we expect 10 ExampleEvent0s and 5 ExampleEvent1s
       const collectedEvent0s = await wallets[0].getPrivateEvents<ExampleEvent0>(
+        testLogContract.address,
         TestLogContract.events.ExampleEvent0,
         firstBlockNumber,
         numBlocks,
+        [wallets[0].getAddress(), wallets[1].getAddress()],
       );
 
-      const collectedEvent0sWithIncoming = await wallets[0].getPrivateEvents<ExampleEvent0>(
-        TestLogContract.events.ExampleEvent0,
-        firstBlockNumber,
-        numBlocks,
-        // This function can be called specifying the viewing public keys associated with the encrypted event.
-        [wallets[0].getCompleteAddress().publicKeys.masterIncomingViewingPublicKey],
-      );
-
-      const collectedEvent1s = await wallets[0].getPrivateEvents<ExampleEvent1>(
-        TestLogContract.events.ExampleEvent1,
-        firstBlockNumber,
-        numBlocks,
-        [wallets[0].getCompleteAddress().publicKeys.masterIncomingViewingPublicKey],
-      );
-
-      expect(collectedEvent0sWithIncoming.length).toBe(5);
-      expect(collectedEvent0s.length).toBe(5);
-      expect(collectedEvent1s.length).toBe(5);
-
-      const emptyEvent1s = await wallets[0].getPrivateEvents<ExampleEvent1>(
-        TestLogContract.events.ExampleEvent1,
-        firstBlockNumber,
-        numBlocks,
-        [wallets[0].getCompleteAddress().publicKeys.masterOutgoingViewingPublicKey],
-      );
-
-      expect(emptyEvent1s.length).toBe(0);
+      // TODO(benesjan): Disabled emitting of ExampleEvent1 for now in the contracts it interferes with processing
```

Comment:
This is the broken test with the not-yet-identified bug as described in the comment in the contract above.